styles.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap");
+
+:root {
+  --bg-color: hsl(49 37% 94%);
+  --typewriterspeed: 7s;
+  --typingdelay: 2s;
+}
+
+body {
+  min-height: 100vh;
+  padding: 0;
```

Comment:
group similar properties

Reply:
okay, So it should be ?
`body {
min-height: 100vh;

padding: 0
margin: ;

border: x;}`

Reply:
yes